fdroid/config.yml

```
@@ -0,0 +1,18 @@
+---
+repo_url: https://mobileapp.bitwarden.com/fdroid/repo
+servergitmirrors: https://raw.githubusercontent.com/bitwarden/f-droid/main/fdroid/repo
```

`servergitmirrors` expects `https://github.com/bitwarden/f-droid`, then `fdroid update` derives the `raw.githubusercontent.com` url from it **assuming the default branch is `master`**. 🤦  

When we put the `raw.githubusercontent.com` url here, it is ignored because it's technically invalid. Meaning the mirror is not included. We won't be able to use this property unless we find a way to override the assumed default branch. 😠 

We can use `mirrors` instead
```suggestion
mirrors:
  - https://raw.githubusercontent.com/bitwarden/f-droid/main/fdroid
```